File: resc/WebApplication/src/modules/application/models/application-termination.dto.ts
import { AttachedFile } from "src/infrastructure/models/attached-file.model";
import { NomenclatureDto } from "src/modules/nomenclature/common/models/nomenclature-dto";

export class ApplicationTerminationDto {
  terminationDate: Date;
  terminationReason: NomenclatureDto;
  annexFile: AttachedFile;
}

File: resc/Resc.Application/Applications/Dtos/Modification/ApplicationTerminationDto.cs
using FileStorageNetCore.Models;
using Resc.Application.Nomenclatures.Dtos;
using Resc.Data.Nomenclatures;
using System;

namespace Resc.Application.Applications.Dtos.Modification
{
	public class ApplicationTerminationDto
	{
		public DateTime TerminationDate { get; set; }

		public NomenclatureDto<TerminationReason> TerminationReason { get; set; }

		public AttachedFile AnnexFile { get; set; }
	}
}
